src/cgame/cg_event.cpp

```diff
@@ -58,15 +58,16 @@ static const struct {
 	{ "[lcannon]",     false, true,  TEAM_HUMANS }, // splash
 	{ "[flamer]",      false, true,  TEAM_HUMANS },
 	{ "[flamer]",      false, true,  TEAM_HUMANS }, // splash
-	{ "[flamer]",      false, true,  TEAM_HUMANS }, // burn
+	{ "[fire]",        false, true,  TEAM_HUMANS }, // burn
 	{ "[grenade]",     false, true,  TEAM_HUMANS },
 	{ "[firebomb]",    false, true,  TEAM_HUMANS },
-	{ "crushed",       true,  false, TEAM_NONE   }, // weight (H) // FIXME
```

Comment:
Why is this team_humans now?

Reply:
Because this is for `MOD_WEIGHT_H` aka crushed by human player. See line 68 for `MOD_CRUSH` and line 86 for `MOD_WEIGHT_A`.